S/SPRAL/build_tarballs.jl

```diff
@@ -3,37 +3,44 @@
 using BinaryBuilder, Pkg
 
 name = "SPRAL"
-version = v"0.1.0"
+version = v"2023.03.29"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/ralna/spral.git", "cd2d2e817275f16d586cf72767631b3c7472ce02")
+    GitSource("https://github.com/ralna/spral.git", "b1d6a7409d41d575bff7106a4e9d9a60b8935c2e")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd ${WORKSPACE}/srcdir/spral
+
+if [[ "${target}" == *mingw* ]]; then
+  BLAS_LAPACK="-L${libdir} -lblastrampoline-5"
+else
+  BLAS_LAPACK="-L${libdir} -lblastrampoline"
+fi
+
 if [[ "${target}" == *-freebsd* ]] || [[ "${target}" == *-apple-* ]]; then
     CC=gcc
     CXX=g++
 fi
+
 ./autogen.sh
 mkdir build
 cd build
-CFLAGS=-fPIC CPPFLAGS=-fPIC CXXFLAGS=-fPIC FFLAGS=-fPIC FCFLAGS=-fPIC \
+CFLAGS="-O3 -fPIC" CPPFLAGS="-O3 -fPIC" CXXFLAGS="-O3 -fPIC" FFLAGS="-O3 -fPIC" FCFLAGS="-O3 -fPIC" \
     ../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
```

Comment:
I find
```suggestion
export CFLAGS="-O3 -fPIC"
export CXXFLAGS="-O3 -fPIC"
export FFLAGS="-O3 -fPIC"
export FCLAGS="-O3 -fPIC"
../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
```
easier to read.  Also, `-O3 -fPIC` are pointless flags to pass to the preprocessor.

Reply:
> Also, `-O3 -fPIC` are pointless flags to pass to the preprocessor.

Well, given the errors my gut feeling is that they're either using `CPPFLAGS` where they shouldn't, or they aren't using `CFLAGS`/`CXXFLAGS` where they should :roll_eyes:

Reply:
You wrote `FCLAGS` instead of `FCFLAGS` in your suggestion. :wink:

Reply:
Well, that'd also do it!